src/ckan_to_bigquery.py

```diff
@@ -70,34 +81,84 @@ def search_sql(self, sql):
         '''
         NB: table must be full table id ...
         '''
-        # limit the number of results to ckan.datastore.search.rows_max
+        # limit the number of results to ckan.datastore.search.rows_max + 1
+        # (the +1 is so that we know if the results went over the limit or not)
         try:
              # check rows_max parameter set in CKAN config,
              # while testing as microlibrary (not as ckan ext) ckan is unknown
-            from ckan.common import config
             rows_max = int(config.get('ckan.datastore.search.rows_max', 32000))
         except:
-            rows_max = 32000 # set default rows limit 
-        
+            rows_max = 32000 # set default rows limit  
```

Comment:
This value should be configurable by environment variable.

Reply:
new commit with relevant changes https://github.com/datopian/ckanext-datastore-bigquery/pull/2/commits/4401f501a5a71f1765b6b056138a1794b6ee98ee